### ISSUE TITLE
unit tests: factory-boy>=3.0 breaking change

### ISSUE THREAD

**Issue by dmsimard** [ansible-community/ara#155]
## What component is this about ?

unit tests (i.e, ``tox -e py3``)

## What is happening ?

factory-boy just released a new version and it looks like we might need to adjust unit tests due to breaking changes.

![Screenshot from 2020-08-13 10-36-48](https://user-images.githubusercontent.com/1291204/90148208-14d7d500-dd51-11ea-8d7c-4da1d4a68b19.png)

```
======================================================================
ERROR: api.tests.tests_file (unittest.loader._FailedTest)
----------------------------------------------------------------------
ImportError: Failed to import test module: api.tests.tests_file
Traceback (most recent call last):
  File "/usr/lib64/python3.8/unittest/loader.py", line 436, in _find_test_path
    module = self._get_module_from_name(name)
  File "/usr/lib64/python3.8/unittest/loader.py", line 377, in _get_module_from_name
    __import__(name)
  File "/home/dmsimard/dev/git/ansible-community/ara/ara/api/tests/tests_file.py", line 23, in <module>
    from ara.api.tests import factories, utils
  File "/home/dmsimard/dev/git/ansible-community/ara/ara/api/tests/factories.py", line 36, in <module>
    class PlaybookFactory(factory.DjangoModelFactory):
AttributeError: module 'factory' has no attribute 'DjangoModelFactory'
```

## What should be happening ?

Unit tests should not be failing :)

**Comment by dmsimard** [comment]
Pinned for now: https://review.opendev.org/#/c/746145/

**Comment by dmsimard** [comment]
Hitting this when building the rpm package with an up-to-date factory-boy in Fedora -- might need a fix for this in 1.5...

**Comment by dmsimard** [comment]
Ok so it turns out the fix was simple after all -- the import just changed location as per the release notes: https://github.com/FactoryBoy/factory_boy/blob/master/docs/changelog.rst#300-2020-08-12

I've submitted a fix: https://review.opendev.org/#/c/752260/
